### PR TITLE
[No ticket] Add react-testing-library

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add react testing library to test in react

https://github.com/kentcdodds/react-testing-library

-Add one sample test to test the library

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,14 @@
   "dependencies": {
     "eclint": "^2.8.1",
     "g": "^2.0.1",
+    "jest-dom": "^3.0.0",
```
[the remark this answers is not in the document]
Comment (fcoronelmakingsense) [reply package.json]: in the documentation page they install jest-dom when add react-testing-library too
image

[the remark this answers is not in the document]
Comment (cbernat) [reply package.json]: @andresmoschini for DOM related testing, you can either use jest + enzyme or jest + react resting library.

[the remark this answers is not in the document]
Comment (andresmoschini) [reply package.json]: Mmmm, maybe there are two different tutorials. If I am not wrong, some tests were already running and I prefer to install fewer dependencies.

@mvillaseco what do you think?

File: src/App.js [hunk 2]
```diff
@@ -102,17 +101,7 @@ class App extends Component {
         <header className="App-header">
           {this.state.user ? <HeaderNav user={this.state.user} /> : null}
           <img src={logo} className="App-logo" alt="logo" />
-          <p>
```
Comment (cbernat) [inline src/App.js]: Why did you remove this internalization example from the main page?

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (andresmoschini) [reply package.json]: > @andresmoschini for DOM related testing, you can either use jest + enzyme or jest + react resting library.

If I am not wrong, @lucianaDilizio already tested DOM without installing other libraries.

[the remark this answers is not in the document]
Comment (andresmoschini) [reply package.json]: Please, remember that we are not creating a project from the ground, we are using a boilerplate and helper, it already has a lot of dependencies.

https://github.com/FromDoppler/doppler-webapp/blob/master/package.json#L13

https://github.com/FromDoppler/doppler-webapp/blob/master/yarn.lock#L8461

File: src/App.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/App.js]: i update this pr fixing this

File: package.json [hunk 3]
```diff
@@ -5,12 +5,14 @@
...
     "jwt-decode": "^2.2.0",
     "prettier-check": "^2.0.0",
     "react": "^16.6.3",
     "react-dom": "^16.6.3",
     "react-intl": "^2.7.2",
-    "react-scripts": "2.1.1"
+    "react-scripts": "2.1.1",
+    "react-testing-library": "^5.4.4"
```
Comment (andresmoschini) [inline package.json]: In the end, was this dependency required? What feature depends on it?

File: src/App.test.js [hunk 4]
```diff
@@ -10,15 +15,11 @@ const messages = {
...
 };
 
-const language = 'en';
-
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(
-    <IntlProvider locale={language} messages={messages[language]}>
+it('renders welcome message', () => {
+  const { getByText } = render(
```
Comment (andresmoschini) [inline src/App.test.js]: Sorry, I am not very familiar with ES6 notation, could you explain me in English or Spanish words what is the meaning of this line?
